content/docs/tasks/security/auth-sds/index.md

```diff
@@ -49,14 +49,7 @@ This approach has the following benefits:
 
 * Set up Istio by following the instructions using
   [Helm](/docs/setup/kubernetes/install/helm/) with SDS setup and global mutual
-  TLS enabled:
```

Comment:
why remove this ?

Reply:
We're already pointing the users to the installation page, I don't think we need to list out the commands here as well. This is to prevent out of date information on the sds task page when the installation page gets updated.

Reply:
Not sure if I understand your comment, the installation has that information as well, if you're talking about this command.
`helm template install/kubernetes/helm/istio --name istio --namespace istio-system \
    --values install/kubernetes/helm/istio/values-istio-sds-auth.yaml | kubectl apply -f -`

The `useNormalJwt` and `useTrustworthyJwt` part are just a caveat section if the users do not want to enable trustworthy jwts. They still need to run the helm command.

Reply:
I see, thanks.